bundles/core/src/main/java/com/adobe/cq/commerce/demandware/preview/StaticContentProxyServlet.java

```diff
@@ -52,18 +52,18 @@
     private static final Logger LOG = LoggerFactory.getLogger(StaticContentProxyServlet.class);
 
     @Reference
-    private DemandwareClient demandwareClient;
+    private DemandwareClientProvider clientProvider;
 
     @Override
     protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse response) throws ServletException,
             IOException {
 
         RequestPathInfo pathInfo = request.getRequestPathInfo();
         LOG.debug("Proxy static content for {}", pathInfo.toString());
-        final String remoteUri = DemandwareClient.DEFAULT_SCHEMA + demandwareClient.getEndpoint() + pathInfo.getResourcePath() +
+        final String remoteUri = DemandwareClient.DEFAULT_SCHEMA + clientProvider.getDefaultClient().getEndpoint() + pathInfo.getResourcePath() +
```

Comment:
we have probably two instances here.. and maybe not even a default one. this would mean, we cannot preview on a multi-instance setup.